src/emailpro/account/add/emailpro-account-add.controller.js

```diff
@@ -191,6 +203,16 @@ angular.module('Module.emailpro.controllers')
     $scope.loadOrderList = function () {
       EmailPro.getOrderList($stateParams.productId).then((data) => {
         $scope.ordersList = data;
+        let orderIndex;
+        for (orderIndex = 0; orderIndex < $scope.ordersList.length; orderIndex += 1) {
```

Comment:
`_.forEach` or `_.map` could fit your needs no ?

Reply:
When I use _.forEach or _.map, eslints gives me an error saying I am modifying the function parameters. Hence I am using the traditional for loop.

Do you think I can override ESLint rule here?

Reply:
ESLint gives an error on `data` ? If so you can still store a copy of parameter, and boucle on it, you'll be able to use lodash methods so.

Reply:
You don't have to update ESLint rule.

These options can help you:
- `_.set()` (see: https://lodash.com/docs/3.10.1#set)
- `_.clone()` (see: https://lodash.com/docs/3.10.1#clone)

Reply:
`_.set` will still overwrite the parameter though, it's the same as disabling the rule, it's just that ESLint can't detect that you are not respecting the rule 😄 .

It's why I prefer `_.clone`, even though the performances are worse

Reply:
I have changed the code to use _.map with _.cloneDeep :)

Let me know if you have a better solution to this.